explore/src/main/scala/explore/constraints/TimingWindowsPanel.scala

```diff
@@ -60,6 +60,8 @@ import java.time.Duration
 import java.time.ZonedDateTime
 import scala.concurrent.duration.FiniteDuration
 
+import scalajs.js.timers
```

Comment:
Is this being used?

Reply:
It is being used when the code in the `n times` textbox is selected.